### DISCUSSION TITLE
Where do I find API documentation for $refs stuff?

### DISCUSSION THREAD

**Discussion by starleaf1** [vuetifyjs/vuetify#12740] [General]
Hello.

From this page https://vuetifyjs.com/en/components/date-pickers/ I found that `VDatePicker` has a `.activePicker` attribute which can be accessed via `$refs['myDatePicker']`, I also found that `VMenu` has `$refs['myMenu'].save()` method. Is there a documentation for these '`$refs` interfaces' on Vuetify?

**Comment by lephro** [comment]
I've also searched for this, but couldnt find it so far. Probably its best to look through the sources and "discover" them youself :)
